include/core/parameters.h

```diff
@@ -1066,6 +1067,7 @@ namespace Parameters
     int                          contact_search_frequency;
     bool                         load_particles_from_file;
     std::string                  particles_file;
+    bool                         enable_extra_verbose_output;
```

Comment:
why not have a regular verbosity variable? I don't understand the need to have a new bool for this.